packages/truffle-core/lib/debug/interpreter.js

```diff
@@ -236,25 +236,22 @@ class DebugInterpreter {
   }
 
   start(terminate) {
-    let prompt;
+    if (terminate === undefined) {
+      return util.promisify(this.start.bind(this))();
```

Comment:
Would you mind explaining what's going on here?

Reply:
Sure: `terminate` is a callback that happens to kill the process. This change makes it so that if you don't pass the callback, it just recurses into a promisified version of itself, thus returning a Promise.